### PR TITLE
Template showing batch completion

### PR DESCRIPTION
**Pull request by doinaklinger**
The template shows how to run a monthly operation(batch) to synchronise the leads of two Salesforce accounts.
When the batch completes, an email is sent with the details about batch: its status, whether it has completed, stopped or timed out, the number of records processed, the number of succesful, failure and canceled records, the start and end time of the batch and the batch id to be used for problem determination when searching the logs.
The templates also shows how to check for errors on the status of the batch and create a Slack message in this case.

### CODE REVIEW

File: resources/template-metadata.json [hunk 1]
```diff
@@ -313,5 +313,23 @@
     "targetApps": ["gmail"],
     "tags": ["eventbrite", "gmail", "marketing", "events", "email"],
     "offerings": ["app connect professional"]
-  }]
+  },{
```
Comment (apendrich) [inline resources/template-metadata.json]: Looks like the new stanza has been added twice to the file.

File: resources/markdown/Sync Salesforce Leads between accounts and check and record state_instructions.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+1. Clicking **Create flow** takes you to the App Connect flow editor.
```
Comment (apendrich) [inline resources/markdown/Sync Salesforce Leads between accounts and check and record state_instructions.md]: We've just updated the format of the instructions, so please replace this with:

To refer to these instructions while editing the flow, open [the github page](https://github.com/ot4i/app-connect-templates/blob/master/resources/markdown/Sync%20Salesforce%20Leads%20between%20accounts%20and%20check%20and%20record%20state_instructions.md) (opens in a new window).

1. Click **Create flow** to start using the template.
1. Connect to your source [Salesforce account](https://developer.ibm.com/integration/docs/app-connect/how-to-guides-for-apps/use-ibm-app-connect-salesforce/).
1. Connect to your target Salesforce account.
1. Connect to your Gmail account and update the email subject and body to meet your business needs. 
1. Connect to your [Slack account](https://developer.ibm.com/integration/docs/app-connect/how-to-guides-for-apps/use-ibm-app-connect-slack/) and select a channel where you want batch level error messages to be sent.
1. To start the flow, in the banner open the options menu [&#8942;] then click **Start flow**.

File: resources/template-metadata.json [hunk 3]
```diff
@@ -313,5 +313,23 @@
...
+    "name": "Sync Salesforce Leads between accounts and check and record state",
+    "description": "This template schedules a monthly synchronisation(batch) between the Leads of two Salesforce accounts. When it completes, a Gmail message is sent with the results of the operation.  You can use this template to perform check the status of a batch.",
```
Comment (apendrich) [inline resources/template-metadata.json]: There's a missing space between synchronisation and (batch).

File: resources/template-metadata.json [hunk 4]
```diff
@@ -313,5 +313,23 @@
...
+    "summary": "Scheduled Salesforce to Salesforce and Gmail and slack",
```
Comment (apendrich) [inline resources/template-metadata.json]: The summary needs to be 'Salesforce to 2 applications'

**Comment by doinaklinger** [comment]
Of course @apendrich . Thank you.

**Pull request merged**